.travis.yml

```diff
@@ -0,0 +1,54 @@
+language: python
+sudo: false
+env:
+  - python=2.7
+    iris=master
+    grib_api=1.14.7
+
+#  - python=3.4
+#    iris=master
+#    grib_api=1.14.7
+
+#  - python=3.5
+#    iris=master
+#    grib_api=1.14.7
+
+
+install:
+  # Fetch and install conda
+  # -----------------------
+  - export CONDA_BASE=http://repo.continuum.io/miniconda/Miniconda3
+  - wget ${CONDA_BASE}-latest-Linux-x86_64.sh -O miniconda.sh;
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set show_channel_urls True
+  - conda config --add channels scitools
+  - conda config --add channels conda-forge
+
+  # Install iris-grib's dependencies
+  # --------------------------------
+  - export IRIS_DEPS="biggus numpy cf_units setuptools pyke netcdf4 scipy cartopy"
+  - conda install --quiet --yes python=${python} pip nomkl ${IRIS_DEPS} python-ecmwf_grib=${grib_api}
```

Comment:
Let's install iris (+ hence its dependencies) with conda, with an option to uninstall and install from source instead.